src/App.js

```diff
@@ -40,18 +41,38 @@ function App() {
   }
 
   return (
-    <>
-      <header style={{ textAlign: "center" }}>
-        <h1>Todo List</h1>
-      </header>
-      <hr />
-      <AddTodoForm onAddTodo={addTodo} />
+    <BrowserRouter>
+      <Routes>
+        <Route
+          path='/'
+          element={
+            <>
+              <header style={{ textAlign: "center" }}>
+                <h1>Todo List</h1>
+              </header>
+              <hr />
+            </>
+          }
+        />
```

Comment:
This attribute took me awhile to figure out 😆. Nice job! 💯

Reply:
:laughing: prettier made it look so not pretty!

Reply:
Oh so pretty 😆